tests/app/core/models/session-model.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { assertSessionDataIntegrity } from '../../../../src/app/core/models/session.model';
+import { of } from 'rxjs';
+
+const validSessionData = {
+  openPhases: [
+    'phaseBugReporting',
+    'phaseTeamResponse',
+    'phaseTesterResponse',
+    'phaseModeration',
+  ],
+  phaseBugReporting: 'bugreporting',
+  phaseTeamResponse: 'pe-results',
+  phaseTesterResponse: 'testerresponse',
+  phaseModeration: 'pe-evaluation',
+};
+
+describe('Session Model', () => {
+  describe('Assert Session Data Integrity', () => {
+    it('should throw error on unavailable session', () => {
+      of(undefined)
+        .pipe(assertSessionDataIntegrity())
+        .subscribe({
+          error: (err) =>
+            expect(err).toEqual(new Error('Session Data Unavailable')),
+        });
+    });
+    it('should throw error on incorrect session data', () => {
```

Comment:
```suggestion
    it('should throw error on session data with missing values', () => {
```

Better to mention the specific problem, rather than "incorrect" (all the other negative tests also have "incorrect" data)